internal/runner/jest.go

```diff
@@ -128,7 +128,7 @@ func (j Jest) Run(result *RunResult, testCases []plan.TestCase, retry bool) erro
 			testCase := plan.TestCase{
 				Name:  example.Title,
 				Scope: strings.Join(example.AncestorTitles, " "),
-				Path:  fmt.Sprintf("%s:%d:%d", testPath, example.Location.Line, example.Location.Column),
+				Path:  testPath,
```

Comment:
👀 I have a bit concern over this causing mismatch between JS Test Collector and bktec, I might need to pick your brain on this topic 🙏🏿

Reply:
All good, I understand now `Path` is only used internally by bktec.